### PR TITLE
hackday - add hack for temp onError override

### PR DESCRIPTION
**Pull request by haislip**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -343,13 +344,22 @@ function addNextIdMethods(chunks: Code[]): void {
   `);
 }
 
+function addTemporaryOverrides(config: Config, chunks: Code[]): void {
+  if (config.includeOnErrorOverride) {
+    chunks.push(code`
+    defaultOptions.onError = (e: any) => { const { baseOnError } = require("src/utils/baseOnError"); baseOnError(e) }
```
Comment (bdow) [inline src/index.ts:350]: I'm thinking we might want to conditionally call this, or do some sort of try/catch. If we use this same code-gen in HOP, then its going to fail as that `baseOnError` file and function do not exist.

Comment (bdow) [reply src/index.ts:350]: Oh nvm. I see this is optionally included. So probably fine.

**Review by bdow — APPROVED** [review]
Cool. 🚀

File: src/index.ts (hunk 1 above)
Comment (haislip) [reply src/index.ts:350]: Yea, that's why this is conditionally called above with the "includeOnErrorOverride"... the real fix will be to be able to configure the function call in the config, but that should really be handled in the open source code where defaultOptions is actually defined.  Adding a non-string in that config isn't supported yet, so this is a short-term hack to get it working while I figure out how to approach that side of it

**Pull request merged**